### PR TITLE
Added comment section to 2024 festival page

### PR DESCRIPTION
**Pull request by julianpistorius**
Using: https://cactus.chat/

Currently allows 'guest' access. Can change to requiring an account if abuse becomes a problem.

Screenshot_2024-05-14_23-06-34-resbaz-festival-comments

### CODE REVIEW

**Comment by julianpistorius** [comment]
Can preview the changes here (until my dev environment times out): https://urban-space-system-9vrv74rqg3p6r5-5782.app.github.dev/resbaz/resbazTucson2024.html#comments

(@alex-r-bigelow - It would be great if we could deploy to staging environments for PRs...)

**Comment by val-pf** [comment]
it works for me (I made those two test comments). What would be the point of this, if it is on the full ResBaz2024 website? Would it allow for chatter? How do we make it not overwhelmingly long?

**Review by val-pf — APPROVED** [review]
I don't fully grasp the technical implementation, @alex-r-bigelow may have more weight here. Approve principle idea and simplicity of solution

**Comment by julianpistorius** [comment]
@val-pf I think it only shows the most recent 10 comments (`pageSize`): https://cactus.chat/docs/client/introduction/

You can also see the comments as a Gitter/Matrix room: https://matrix.to/#/%23comments_resbazaz_festival2024%3Acactus.chat

**Comment by julianpistorius** [comment]
Oh, and I forgot to mention that there is a moderation feature - also done through Matrix.

File: resbaz/resbazTucson2024.qmd [hunk 1]
```diff
@@ -96,3 +100,20 @@ We are committed to making this workshop accessible to everybody. For any accomm
...
+initComments({
+  node: document.getElementById("comment-section"),
+  defaultHomeserverUrl: "https://matrix.cactus.chat:8448",
+  serverName: "cactus.chat",
+  siteName: "resbazaz",
+  commentSectionId: "festival2024",
+  updateInterval: 15,
+  loginEnabled: true,
+  guestPostingEnabled: true
+})
```
Comment (alex-r-bigelow) [inline resbaz/resbazTucson2024.qmd:118]:
```suggestion
  guestPostingEnabled: true,
  pageSize: 6
})
```

**Review by alex-r-bigelow — APPROVED** [review]
A thousand apologies for my late response; connectivity issues today

This looks great!

I imagine that `guestPostingEnabled` is a thing that's only cosmetic, and *really* needs to be set from the Cactus perspective to actually protect anything? i.e. anyone could change that setting in their browser if they wanted to, even if we turned it off...

**Comment by julianpistorius** [comment]
Thanks for the help @alex-r-bigelow!

**Pull request merged**